experiment/bigquery/flakes.sh

```diff
@@ -25,12 +25,12 @@ if [[ ! -f "${out}" ]]; then
   cat "$(dirname "${0}")/flakes.sql" | bq query --format=prettyjson > "${out}"
 fi
 which jq >/dev/null || (echo 'Cannot find jq on path. Install jq' 1>&2 && exit 1)
-echo 'Jobs flaking more than 3x/day:' 1>&2
+echo 'Jobs flaking more than 2x/day:' 1>&2
```

Comment:
What enforces the 2 flakes/day rule? 3 flakes per day was enforced by checking (.flakes|tonumber > 21) but that filter was removed.